### PR TITLE
AD Kibana IntegTest addition to build workflows WITH Security

### PR DESCRIPTION
**Pull request by peterzhuamazon**
This PR is to add the AD Kibana IntegTest addition to build workflows WITH Security with #333.

### Test Cases: ###

~~>NOTE: Since the infra workflows are pulling tagged versions and the recent changes are not part of any tags, it will obviously fail until we push a new tag.~~
I am testing with master branch for AD Kibana Repo now, hash 9e464623773e6cfc500d3abe5b6b06a630d68c83

* TAR: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/213896814
* DEB: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/214138954
* RPM: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/214154505
* DOCKER: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/214487428
* WINDOWS: https://github.com/opendistro-for-elasticsearch/opendistro-build/actions/runs/214337142

### CODE REVIEW

**Comment by gaiksaya** [comment]
You can run test against the master for testing purpose just to make sure there is no issue from infra side. I did the same for alerting.

**Comment by peterzhuamazon** [comment]
> You can run test against the master for testing purpose just to make sure there is no issue from infra side. I did the same for alerting.

I am pulling the latest changes from master.

**Comment by weicongs-amazon** [comment]
> > You can run test against the master for testing purpose just to make sure there is no issue from infra side. I did the same for alerting.
> 
> ~I am pulling the latest changes from master.~
> Oh, you are talking about the tags, doesnt matter.
> We just make sure the steps are there, and it should fail for 1.9.0.0, rest is from plugin teams' side to have another tag during next release.
This is different. We need make sure our logic work at least once. Last time, for kibana plugin, it took me long time to get logic right.  It's better to use master branch to run tests. Test case error is ok, but the test case should be executed.

**Comment by peterzhuamazon** [comment]
> > > You can run test against the master for testing purpose just to make sure there is no issue from infra side. I did the same for alerting.
> > 
> > 
> > ~I am pulling the latest changes from master.~
> > Oh, you are talking about the tags, doesnt matter.
> > We just make sure the steps are there, and it should fail for 1.9.0.0, rest is from plugin teams' side to have another tag during next release.
> 
> This is different. We need make sure our logic work at least once. Last time, for kibana plugin, it took me long time to get logic right. It's better to use master branch to run tests. Test case error is ok, but the test case should be executed.

Will do.

File: .github/workflows/staging-build-tar-temp.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+name: Process TAR Artifacts
```
Comment (camerski) [inline .github/workflows/staging-build-tar-temp.yml:1]: It looks like staging-build-tar-temp.yml duplicates a lot of the changes to staging-build-tar.yml. Is that intentional?

Comment (peterzhuamazon) [reply .github/workflows/staging-build-tar-temp.yml:1]: Testing file, will remove once everything is ready.

File: .github/scripts/setup_runners.sh [hunk 2]
```diff
@@ -68,7 +68,7 @@ fi
 SETUP_ACTION=$1
 SETUP_INSTANCE=`echo $2 | sed 's/,/ /g'`
 SETUP_TOKEN=$3
-SETUP_AMI_ID="ami-0655d7056d3941cd2"
+SETUP_AMI_ID="ami-042f29d2ac35db697"
```
Comment (camerski) [inline .github/scripts/setup_runners.sh:71]: Just out of interest, what changed in the AMI?

File: .github/scripts/setup_runners_service.sh [hunk 3]
```diff
@@ -240,7 +238,7 @@ then
   elif [ "$SETUP_DISTRO" = "docker" ]
   then
     docker restart $DOCKER_NAME
-    docker run -d -p 5601:5601 --name $DOCKER_NAME_KIBANA --network="host" odfe-kibana-http:security
+    docker run -d --name $DOCKER_NAME_KIBANA --network="host" odfe-kibana-http:security
```
Comment (camerski) [inline .github/scripts/setup_runners_service.sh:241]: Does it still work without exposing the Kibana port? If so, how?

File: .github/scripts/setup_runners.sh (hunk 2 above)
Comment (peterzhuamazon) [reply .github/scripts/setup_runners.sh:71]: GitHub release a new runner binary out of nowhere and it does not have bc so all runners broke.
Add the python3 installation as well.
If this happens again I would consider making it more dynamic, but they also didnt update broken change in months.

File: .github/scripts/setup_runners_service.sh (hunk 3 above)
Comment (peterzhuamazon) [reply .github/scripts/setup_runners_service.sh:241]: You should not / need not to use port binding anymore if you use host network.

Comment (camerski) [reply .github/scripts/setup_runners_service.sh:241]: OK, cool.

**Pull request merged**